### ISSUE TITLE
24-word seed phrase support was dropped

### ISSUE THREAD

**Issue by aolieman** [stellar/freighter#980]
### What version are you using?
5.5.0

### What did you do?
I attempted to recover my wallet in a new Firefox instance (e.g. new laptop). My Freighter wallet includes multiple accounts and is based on a 24-word seed phrase. This wallet was originally migrated from xBull (hence the 24 words), but has since added accounts that only Freighter knows about.

### What did you expect to see?
A way to input a 24 word recovery phrase.

### What did you see instead?
Dedicated inputs for 12 words. An older Freighter version allowed me to enter an arbitrary amount of words in a single text field.

I have tried installing the older version, but the earliest available as a Firefox addon is version 4.0.2, which already has the dedicated 12 input fields. Do I need to install an older version from GitHub to proceed? It would be very nice to have a way to recover from 24 word phrases in the latest release.

**Comment by aolieman** [comment]
There is a workaround available for stuck users who are somewhat comfortable with JS. @piyalbasu helped by pointing me at the relevant key derivation library, https://www.npmjs.com/package/stellar-hd-wallet.

Assuming a recent node + npm installation:

1. First `npm install stellar-hd-wallet`
2. Run the node REPL simply by executing `node` in a terminal without any parameters
3. Then, in the REPL:

```
const { default: StellarHDWallet } = await import("stellar-hd-wallet");
const wallet = StellarHDWallet.fromMnemonic('<your recovery phrase>')
const keypair = wallet.getKeypair(0)  // depending on the path, it was 1 in my case
keypair.publicKey()  // to check if it's the account you want to recover
keypair.secret()  // prints out the secret key
```

This secret can be loaded into the latest version of Freighter, but only after creating a new wallet or restoring from a 12 word phrase. The option to import a secret key is listed in the account switching menu, accessed by clicking on the account "avatar" after the wallet is fully initialized.

**Comment by piyalbasu** [comment]
Thanks for the feedback. This has been added to our roadmap and will be implemented in the near future!

**Comment by piyalbasu** [comment]
Importing 24 word mnemonic phrases is now live in version 5.13.0: https://github.com/stellar/freighter/releases/tag/5.13.0

**Comment by aolieman** [comment]
Excellent, I was able to test it and the 24-word recovery works as expected. Thank you!
